### DISCUSSION TITLE
Date picker, time, and date columns

### DISCUSSION THREAD

**Discussion by hjbout** [Budibase/budibase#5284] [Help] [answered]
Hi,

I'm struggling with getting my dates and date pickers time in the right format. I want them in the dutch formats; dd-mm-yyyy and 24-hours times. For (table) columns I can do the formatting I guess (albeit one by one). But the times in the date picker I can't get them to show in 24-hour format. Is this even possible right now? It is a kind of a "deal-breaker" for me ...

I tried setting the locales in Debian and setting the browser language but to no avail. 
Any tip is welcome :)

**Comment by melohagan** [comment, marked as answer]
Hi @hjbout 

At the moment there is no option for choosing 24-Hour time within the date picker. 
It is a good idea, and I've raised a feature request here: https://github.com/Budibase/budibase/issues/5287

Thanks for your interest in budibase!

**Reply by hjbout** [reply]
Thanks @melohagan
